prometheus_es_exporter/__init__.py

```diff
@@ -354,11 +360,18 @@ def main():
         level=logging.DEBUG if args.verbose else log_level
     )
     logging.captureWarnings(True)
-
     port = args.port
     es_cluster = args.es_cluster.split(',')
 
-    if args.ca_certs:
+    if args.client_cert and args.client_key and args.ca_certs:
+        es_client = Elasticsearch(es_cluster,
+                                  verify_certs=True,
+                                  scheme="https",
```

Comment:
Is this to force HTTPS to be used? I haven't checked the code, but couldn't spot this parameter in the documentation - just `use_ssl`.

I think I'd prefer to leave it up to the URLs in `es_cluser` to determine if HTTPS should be used for a given node, rather than forcing it here,